Bounding_volumes/doc/Bounding_volumes/CGAL/Min_sphere_of_spheres_d_traits_3.h

```diff
@@ -35,7 +35,7 @@ class Min_sphere_of_spheres_d_traits_3 {
 /*!
 is the constant 3, i.e.\ the dimension of \f$ \mathbb{R}^3\f$. 
 */ 
-typedef unspecified_type D; 
+static const int D;
```

Comment:
Yes, I hesitated to do this, but as the original doc was completed by `is the constant 3`, I thought I'd leave it that way.